lib/db/mysql/schema.sql

```diff
@@ -43,3 +43,14 @@ CREATE TABLE IF NOT EXISTS tokens (
   scope VARCHAR(256) NOT NULL,
   createdAt TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
 ) ENGINE=InnoDB CHARACTER SET utf8 COLLATE utf8_unicode_ci;
+
+CREATE TABLE IF NOT EXISTS developers (
+  email VARCHAR(256) NOT NULL PRIMARY KEY,
```

Comment:
IMHO the simplest approach will be to generate a unique ID in the app and pass it in explicitly when inserting the row.  Using a unique index at the db level would prevent against the very small chance of accidental duplication.

Also, what's the thinking behin 16-versus-32 bytes?